pyvista/examples/downloads.py

```diff
@@ -539,7 +539,7 @@ def download_kitchen(split=False):
         alg.SetInputDataObject(mesh)
         alg.SetExtent(extent)
         alg.Update()
-        result = pyvista.filters._get_output(alg)
+        result = pyvista.core.filters.vtkalgorithm._get_output(alg)
```

Comment:
@JevinJ It seems that building document error occured here. Do you have any idea?

Reply:
I made a vtkalgorithm class at first, but I decided there wasn't a good enough reason for it yet. This should be fixed.